dev-util/dialog/dialog-1.3_20240619.recipe

```diff
@@ -4,18 +4,21 @@ text user interface widgets. It uses the curses or ncurses library."
 HOMEPAGE="https://invisible-island.net/dialog/"
 COPYRIGHT="1999-2014 Thomas E. Dickey"
 LICENSE="GNU LGPL v2"
-SOURCE_URI="ftp://ftp.invisible-island.net/dialog/dialog-${portVersion/_/-}.tgz"
-CHECKSUM_SHA256="2ff1ba74c632b9d13a0d0d2c942295dd4e8909694eeeded7908a467d0bcd4756"
+SOURCE_URI="https://invisible-island.net/archives/dialog/dialog-${portVersion/_/-}.tgz"
+CHECKSUM_SHA256="5d8c4318963db3fd383525340276e0e05ee3dea9a6686c20779f5433b199547d"
 SOURCE_DIR="dialog-${portVersion/_/-}"
-REVISION="3"
+REVISION="1"
 PATCHES="dialog-$portVersion.patchset"
 
 ARCHITECTURES="all"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

Comment:
What could be done for optimization purposes is move the command to the secondaryArch package, but if you want to keep the primaryArch library, you can't use `REPLACES`, so it would create a hassle for people upgrading if nothing depends on `cmd:dialog` specifically, so all in all, it is probably not worth it.

Reply:
I "could" just nuke gcc2 all together (last version bump here is 7 years ago and no one complained yet) :)

Reply:
Resolving this, not putting more time into it, if someone feels the need to make further changes I won't stop them. :)